kubernetes/windows/main.ps1

```diff
@@ -423,7 +423,24 @@ function Read-Configs {
 
     # run mdm config parser
     ruby /opt/amalogswindows/scripts/ruby/tomlparser-mdm-metrics-config.rb
-    .\setmdmenv.ps1
+    Set-EnvironmentVariablesFromFile "/opt/amalogswindows/scripts/powershell/setmdmenv.txt"
+}
+
+function Set-EnvironmentVariablesFromFile {
+    param ($filePath)
+
+    $envVars = @{}
```

Comment:
check if filePath valid and exists, and throw an error if its not valid/exists

Reply:
changed